frontend/webapp/hooks/useKeyDown.tsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from "react";
+
+export function useKeyDown(key, callback) {
+  const handleKeyDown = (event) => {
+    if (key === event.key) {
+      callback(event);
+    }
+  };
+
+  useEffect(() => {
+    window.addEventListener("keydown", handleKeyDown);
```

Comment:
keypress event is a deprecated feature and is no longer recommended. please replace the event with the relevant one

Reply:
keypress is deprecated, so i used keydown instead

![image](https://github.com/keyval-dev/odigos/assets/82360525/088e85e2-166d-4efe-b232-e85b39da741b)